dotCMS/src/main/java/com/dotcms/mail/MailAPIImpl.java

```diff
@@ -37,6 +39,23 @@ public Session getMailSession() {
         return mailSessionHolder.get();
     }
 
+    private String getProtocol(){
+        return this.properties.containsKey(Keys.TRANSPORT_PROTOCOL.getValue())
+                ? this.properties.getProperty(Keys.TRANSPORT_PROTOCOL.getValue()) : "smtp";
+    }
+
+    @Override
+    public int getConnectionPort(){
+        return this.properties.containsKey("mail." + protocol.get() + ".port")?
+                Integer.parseInt(this.properties.getProperty("mail." + protocol.get() + ".port")):25;
```

Comment:
this could be a constant DEFAULT_MAIL_PORT or so

Reply:
Done!